layouts/events/list.html

```diff
@@ -13,7 +13,17 @@ <h1>Upcoming Events</h1>
 
 {{ define "main" }}
     <div class="container mx-auto">
-        <div class="flex justify-between py-10 md:py-24 px-5 md:px-20">
+        <div class="w-10/12 mx-auto mt-10 note note-info" role="alert">
+            <p class="text-lg">
+                <i class="fas fa-info-circle pr-2"></i>
+                The demand for Pulumi content and education continues to grow and, even in
+                the face of restricted travel, we will ramp it up further. Although many events
+                worldwide we’d planned to sponsor or otherwise participate in have been canceled,
+                we have shifted to working with customers through digital channels wherever we can.
+                Checkout our <a href="{{ relref . "/webinars" }}"><b>upcoming webinars</b></a>.
```

Comment:
I'd prefer without the bold, but if we want to make this stand out, we should use `<strong>` instead, 'cause semantic.

Reply:
👍